<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatteryCabinetForm</class>
 <widget class="QWidget" name="BatteryCabinetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="GroupBoxEx" name="groupBoxCtrl">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>70</number>
      </property>
      <item row="0" column="0">
       <widget class="LabelEx" name="label_2">
        <property name="text">
         <string>电池柜编号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditEx" name="lineEditID_Ctrl">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="LabelEx" name="label_12">
        <property name="text">
         <string>电池柜状态：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="ComboBoxEx" name="comboBoxState_Ctrl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="PushButtonEx" name="pushButtonSet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>设  置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="GroupBoxEx" name="groupBoxView">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>70</number>
      </property>
      <item row="0" column="0">
       <widget class="LabelEx" name="label">
        <property name="text">
         <string>电池柜编号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditEx" name="lineEditID">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="LabelEx" name="label_3">
        <property name="text">
         <string>电池柜状态：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="ComboBoxEx" name="comboBoxState">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="LabelEx" name="label_4">
        <property name="text">
         <string>地理位置：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="LineEditEx" name="lineEditLocation">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="9">
       <widget class="PushButtonEx" name="pushButtonReset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>重  置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="LabelEx" name="label_5">
        <property name="text">
         <string>连接状态：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ComboBoxEx" name="comboBoxConnect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="LabelEx" name="label_6">
        <property name="text">
         <string>通信地址：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="LineEditEx" name="lineEditAddress">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="6">
       <widget class="LabelEx" name="label_7">
        <property name="text">
         <string>SIM卡号码：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="LineEditEx" name="lineEditSIM">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="PushButtonEx" name="pushButtonQuery">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>查  询</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="10">
       <widget class="TableViewEx" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButtonEx</class>
   <extends>QPushButton</extends>
   <header>ctrls/PushButtonEx.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>ctrls/ComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>LineEditEx</class>
   <extends>QLineEdit</extends>
   <header>ctrls/LineEditEx.h</header>
  </customwidget>
  <customwidget>
   <class>LabelEx</class>
   <extends>QLabel</extends>
   <header>ctrls/LabelEx.h</header>
  </customwidget>
  <customwidget>
   <class>TableViewEx</class>
   <extends>QTableView</extends>
   <header>ctrls/TableViewEx.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBoxEx</class>
   <extends>QGroupBox</extends>
   <header>ctrls/GroupBoxEx.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
